docs/redis.md

```diff
@@ -64,7 +64,7 @@ We just showed how `tye` makes it easier to communicate between 2 applications r
 
        services.AddStackExchangeRedisCache(o =>
        {
-            var connectionString = Configuration["connectionstring:redis"];
+            var connectionString = Configuration["connectionstrings:redis"];
```

Comment:
Use the extension method.

Reply:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/1430011/78512485-c9396d00-7759-11ea-9884-5d0b97981a81.png">

Reply:
https://github.com/dotnet/tye/pull/299#issuecomment-609501437